VulnerableApplication/Models/Hello.cs

```diff
@@ -11,5 +11,9 @@ public void Say()
         {
             Console.WriteLine("Hello!");
         }
+    public string Hi() {
+    return "this is against the rules.";
```

Comment:
[CODE_SMELL] 2 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.

Reply:
/workitem